app/api/schema/custom_forms.py

```diff
@@ -52,7 +52,7 @@ class Meta:
     is_included = fields.Boolean(default=False)
     is_public = fields.Boolean(default=False)
     position = fields.Integer(allow_none=True, default=0)
-    is_complex = fields.Boolean(dump_only=True)
+    is_complex = fields.Boolean(default=False)
```

Comment:
That is a surprising change. The `dump_only` looks completely out of place, and `default=` looks correct.
How did that come in there?

Reply:
After checked, I think that should be contained both. dump_only and default. I think that should not be modified after created; hence, dump_only still correct. I will add it back.